.circleci/config.yml

```diff
@@ -262,6 +269,56 @@ defaults:
 # -----------------------------------------------------------------------------------------------
 jobs:
 
+  b_windows:
+    executor:
+      name: win/default
+      shell: powershell.exe
+    steps:
+      - checkout
+      - run: systeminfo
+      - run:
+          name: "Download CMake"
```

Comment:
Wouldn't it be better to move the first three steps into a script?

Reply:
(Install CMake, Install solidity dependencies, install EVMONE) - I don't see a huge benefit in merging these. But I took a quick look and one problem I see is at least that the `install_deps.bat` is just calling `install_deps.cmake`, whose syntax isn't quite intuitive to look at / understand.

Reply:
My main motivation is to keep the circleci config small. After all, it is a config and not a script.